app/services/update_user.rb

```diff
@@ -0,0 +1,14 @@
+class UpdateUser
```

Comment:
To be able to test the Form class without having to interact with the DB. If all forms called `update!` directly, that would mean interacting with the DB in all form specs to verify that the user was indeed updated, versus having that handled by an abstract class, where the interaction with the DB only happens in that one test. It's also more future proof. If Rails changes the way `update!` works, or removes it, we will only need to update `UpdateUser`, versus all forms where `update!` was called.